tests/python/relay/dyn/test_dynamic_op_level10.py

```diff
@@ -27,34 +27,62 @@
 import random
 import tvm.testing
 
-# TODO(mbrookhart): Enable when VM supports heterogenus execution
+# TODO(mbrookhart): Enable when the VM supports heterogenus execution
```

Comment:
Should we enable this since VM supports heterogeneous execution now? or you plan to enable them all in a followup PR?

Reply:
Still fighting strided slice and topk, I was planning on enabling these with that PR when I get it working.